### PR TITLE
Feat: Payable service & proposal fee on platform requirement

### PR DESCRIPTION
**Pull request by Gathin23**
Screenshot_20230928_121414

### CODE REVIEW

File: src/components/Form/ProposalForm.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { ErrorMessage, Field, Form, Formik } from 'formik';
 import { QuestionMarkCircle } from 'heroicons-react';
```
Comment (0xRomain) [inline src/components/Form/ProposalForm.tsx:2]: Can you add a screenshot to the PR please

File: src/components/Form/ProposalForm.tsx [hunk 2]
```diff
@@ -57,6 +58,10 @@ function ProposalForm({
   const { isActiveDelegate } = useContext(TalentLayerContext);
   const { platformHasAccess } = useContext(Web3MailContext);
   const [aiLoading, setAiLoading] = useState(false);
+  
+  const platform = usePlatform(process.env.NEXT_PUBLIC_PLATFORM_ID as string);
+  const proposalFee = platform?.proposalPostingFee || 0;
```
Comment (0xRomain) [inline src/components/Form/ProposalForm.tsx:63]: to make the naming coherent proposalPosting..

File: src/components/Form/ServiceForm.tsx [hunk 3]
```diff
@@ -211,6 +215,8 @@ function ServiceForm() {
                 <span className='text-red-500 mt-2'>
                   <ErrorMessage name='rateAmount' />
                 </span>
+                {servicePostingFeeFormat !== 0 && (
+                <span className='text-gray-100'>Fee for posting a service: {servicePostingFeeFormat} MATIC</span>)}
```
[the remark this answers is not in the document]
Comment (Gathin23) [reply src/components/Form/ServiceForm.tsx:219]: So even on the admin Fee setting page it should show the name dynamically right?

**Comment by Gathin23** [comment]
@0xRomain 

- I have added a screenshot to the PR
- I have made it to show the symbol name dynamically and also to update the decimals dynamically based on the chain
- Have made the necessary naming changes

**Review by 0xRomain — APPROVED** [review]

**Pull request merged**